Competition/src/main/cpp/Arm.cpp

```diff
@@ -0,0 +1,98 @@
+#include "Arm.h"
+
+Arm::Arm() : armMtrLeft{ArmConstants::TALON_ID_LEFT_ARM}, 
```

Comment:
Not passing in a controller pointer would leave the controller null, and therefore seg fault later in the code when trying to utilize the controller member variable.

Therefore, do not allow the user to use a constructor that doesn't pass in a controller pointer. Delete this implementation and delete the definition in the header file

Reply:
Okay, the only reason I had this is because when I had the GetInstance() method in, it would give me an error saying there was no default constructor for when it did "static Arm instance" but I guess that really isn't a good solution to the problem at all.

Reply:
Ahh that's a good point. I'll need to think about that, cause yeah that's a problem